src/lib/data/nodes_desc.json

```diff
@@ -171,10 +170,8 @@
 		"stats": ["60% increased maximum Energy Shield", "20% slower start of Energy Shield Recharge"]
 	},
 	"N13": {
-		"name": "Electric Amplification",
-		"stats": [
-			"Damage Penetrates 18% Lightning Resistance",
-			"25% increased Lightning Exposure Effect"]
+		"name": "N13",
```

Comment:
are these intentionally reverted? no comment about a lot of these nodes in the PR